.github/workflows/preflight.yml

```diff
@@ -26,14 +29,25 @@ jobs:
       - name: Get go version
         id: go-version
         run: echo "name=version::$(go env GOVERSION)" >> $GITHUB_OUTPUT
+      - name: FLY_PREFLIGHT_TEST_APP_PREFIX
+        run: |
+          echo "FLY_PREFLIGHT_TEST_APP_PREFIX=pf-gha-$(openssl rand -hex 4)" >> "$GITHUB_ENV"
       - name: Run preflight tests
         env:
           FLY_PREFLIGHT_TEST_ACCESS_TOKEN: ${{ secrets.FLYCTL_PREFLIGHT_CI_FLY_API_TOKEN }}
           FLY_PREFLIGHT_TEST_FLY_ORG: flyctl-ci-preflight
           FLY_PREFLIGHT_TEST_FLY_REGIONS: iad syd
-        run: make ci-preflight
+        run: |
+          make
+
+          curl -sfL https://raw.githubusercontent.com/Songmu/gotesplit/v0.2.1/install.sh | sh -s
+
+          if [[ "${{ github.ref }}" != "refs/heads/master" ]]; then
+            test_opts=-short
```

Comment:
will you move this run script this into a real script or a make target? debugging and editing stuff in the github workflow yaml is... challenging :-)

Reply:
Haha. Yeah. Let me extract that as a small shell script.